bundles/org.openhab.binding.octopusenergy/src/main/java/org/openhab/binding/octopusenergy/internal/dto/AccountProperty.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.octopusenergy.internal.dto;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link AccountProperty} is a DTO class representing a physical building/location (e.g. a private house).
+ *
+ * @author Rene Scherer - Initial contribution
+ */
+@NonNullByDefault
+public class AccountProperty {
```

Comment:
DTOs shouldn't be annotated with NonNullByDefault, because it gives a false sense of safety as the JSON serializer could set fields to null.